beacon-chain/blockchain/van_process_pending_blocks.go

```diff
@@ -8,14 +8,15 @@ import (
 	"github.com/prysmaticlabs/prysm/beacon-chain/core/feed"
 	blockfeed "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/block"
 	statefeed "github.com/prysmaticlabs/prysm/beacon-chain/core/feed/state"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
 	"github.com/prysmaticlabs/prysm/proto/interfaces"
 	vanTypes "github.com/prysmaticlabs/prysm/shared/params"
 	"time"
 )
 
 var (
-	// Getting confirmation status from orchestrator after each confirmationStatusFetchingInverval
-	confirmationStatusFetchingInverval = 500 * time.Millisecond
+	// Getting confirmation status from orchestrator after each confirmationStatusFetchingInterval
```

Comment:
Docstring should start with the name
```suggestion
	// confirmationStatusFetchingInterval is the delay between confirmation statuses from orchestrator
```
or somthing like that

Reply:
done